src/uu/mktemp/src/mktemp.rs

```diff
@@ -546,7 +547,22 @@ fn exec(dir: &str, prefix: &str, rand: usize, suffix: &str, make_dir: bool) -> U
     // relative path.
     let path = Path::new(dir).join(filename);
 
-    println_verbatim(path).map_err_context(|| "failed to print directory name".to_owned())
+    Ok(path)
+}
+
+pub fn mktemp(
+    dir: &str,
```

Comment:
Would it be possible to make this into a `&Path`?

Reply:
Yeah sounds good